ontology/yaml/resources/states/states.yaml

```diff
@@ -56,7 +56,9 @@ STANDBY: "Waiting in a prepared state."
 REMOTE: "The control is operated remotely."
 LOCAL: "The control is operating locally and independently with the remote system."
 
-
+#New states for CH ZRH eurd
+FADING: "An operation mode of a lighting dimmer"
```

Comment:
Expand definition to distinguish from ramping. Fading goes from high illuminance to low?